templates/Web/_composition/Flask/Page.Flask.List.AddRoutes/.template.config/template.json

```diff
@@ -10,7 +10,7 @@
     "wts.type": "composition",
     "wts.platform": "Web",
     "wts.version": "1.0.0",
-    "wts.compositionFilter": "$backendframework == Flask & groupIdentity == wts.Page.List"
+    "wts.compositionFilter": "$backendframework == Flask & groupIdentity == wts.Page.List & $feature != wts.Feature.Azure.Cosmos.Mongo & $feature != wts.Feature.Azure.Cosmos.SQL"
```

Comment:
A thought for later: how will this logic scale if we start adding more databases?

Reply:
Good point. It would be a lot of clutter if we are going to use this logic. I am not sure but I think maybe if there is something like '$feature=None` if it even exists